plasma_framework/test/helpers/positions.js

```diff
@@ -1,10 +1,19 @@
 const BLOCK_OFFSET = 1000000000;
 const TX_OFFSET = 10000;
+const BLOCK_OFFSET_FOR_TX_POS = 1000000000 / 10000;
 
 function buildUtxoPos(blockNum, txIndex, outputIndex) {
     return blockNum * BLOCK_OFFSET + txIndex * TX_OFFSET + outputIndex;
 }
 
+function buildTxPos(blockNum, txIndex) {
+    return blockNum * BLOCK_OFFSET_FOR_TX_POS + txIndex;
```

Comment:
Consider moving the `BLOCK_OFFSET_FOR_TX_POS` constant into this code ... in here it makes sense, otherwise it takes time to understand why this constant exists